docs/program/sdks/_index.md

```diff
@@ -254,6 +300,11 @@ go run ~/myCode/myViamFile.py
 
 For an example, see [this execution demo.](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples/vanilla)
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+For an example, see [this execution demo.](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples/vanilla)
```

Comment:
This is the TS example. I wasn't able to guess a likely C++ example from the upstream repo; not sure what this should link to.